packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -75,15 +75,16 @@ export default class SaytDriverPlugin implements Plugin {
   }
 
   /**
-   * Dispatches an event with the response from the sayt data plugin.
+   * Dispatches an event with the response from the sayt data plugin and the associated searchbox ID.
    * Callback for the Sayt data request event listener.
    *
    * @param event Event that contains the Sayt API request payload.
    */
   fetchSaytData(event: CustomEvent): void {
```

Comment:
You can give the payload a type by passing in a type parameter.

```suggestion
  fetchSaytData(event: CustomEvent<AutocompleteRequestConfig>): void {
```